Dumper/TranslationDumper.php

```diff
@@ -207,14 +207,34 @@ private function dumpTranslationsPerDomain($pattern, array $formats, $target)
         }
     }
 
-    private function dumpTranslationsPerLocale($pattern, array $formats, $target)
+    private function dumpTranslationsPerLocale($pattern, array $formats, $target, bool $mergeFallback)
```

Comment:
if the bundle start using scalar typehints, this requires PHP 7.

This is perfectly fine with me, but only if we tell Composer about it (in composer.json). Otherwise, composer will happily install the new version of the bundle on PHP 5.6, meaning that any PHP requirement change has to be considered a very hard BC break (as users will have to handle dependencies by hand)